introduction.tex

```diff
@@ -62,6 +62,9 @@ \subsection{Register Definition Format}
         \hline
         R/W1 & Read/Write. Only writing 1 has an effect.  \\
         \hline
+        R/W1C & Read/Write. Writing every bit high clears the field. Writing
```

Comment:
Well there is one more option which is what is actually implemented in Spike/Rocket-Chip:

4. Each bit is treated like an unrelated W1C field. 

Since our error codes are sticky, the race doesn't really exist, and it's simpler if the update logic only needs to consider 1 bit.